src/components/InputForm/style.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+export const DivInputForm = styled.div`
+  //border: 2px solid blue;
+  margin-bottom: 10px;
+  overflow: auto;
+`;
+
+export const InputStyle = styled.input`
+  color: white;
+  background: none;
+  border: none;
+  border-right: 0px;
+  border-top: 0px;
+  boder-left: 0px;
+  boder-bottom: 0px;
```

Comment:
`border: none;`을 적용해 주셔서 아래 코드는 불필요해 보입니다.

```suggestion
  border: none;
```